scripts/sync-algolia-index.ts

```diff
@@ -91,8 +94,16 @@ const generateAlgoliaRecords: () => AlgoliaRecord[] = () => {
     return appendData;
   };
 
-  const specFiles = getFileInfos("../../../site/src/_pages/spec", [], "spec");
-  const docsFiles = getFileInfos("../../../site/src/_pages/docs", [], "docs");
+  const specFiles = getFileInfos(
+    path.resolve(monorepoDirPath, "site/src/_pages/spec"),
+    [],
+    "spec",
+  );
+  const docsFiles = getFileInfos(
+    path.resolve(monorepoDirPath, "site/src/_pages/docs"),
+    [],
+    "docs",
+  );
```

Comment:
@CiaranMn also fixed paths, which broke when I moved this script:
https://github.com/blockprotocol/blockprotocol/runs/5064081770?check_suite_focus=true